src/Config/Config.php

```diff
@@ -52,4 +52,14 @@ class Config
     public const COLLECTION_WISHLIST_PRODUCTS = 'wishlist_products';
     public const COLLECTION_STOCKS = 'stocks';
     public const COLLECTION_STOCK_MVTS = 'stock_movements';
+
+    /**
+     * @param mixed $message
+     *
+     * @return void
+     */
+    public static function dev_log($message)
+    {
+        file_put_contents('/var/www/html/php.log', $message . PHP_EOL, FILE_APPEND);
```

Comment:
Is there a more common way to push data in the nginx/apache server output?
I guess we should have an env var for this, you may ask the core team